src/App.tsx

```diff
@@ -1,26 +1,199 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { deleteTodo, getTodos, postTodo, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { sendErrorMessage } from './components/errorsUnderFooter';
+import { useRef } from 'react';
+import { Footer } from './components/footer';
+import { ErrorMessage } from './components/errorsUnderFooter';
+import { MainSection } from './components/section';
+import { Header } from './components/header';
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[]>([]);
+  const [filteredTodoList, setFilteredTodoList] = useState<Todo[]>([]);
```

Comment:
Don't need to create additional state for filtered todos, use only one state
```suggestion
  const [todoList, setTodoList] = useState<Todo[]>([]);
```